src/logger.ts

```diff
@@ -1,36 +1,23 @@
 import { CmdGlobalOptions } from "./commands/cmd-options.ts";
 import { isatty } from "./commands/tty.ts";
-import { log } from "./deps.ts";
 
-export async function setupLogger(options: CmdGlobalOptions) {
-  const defaultLogLevel: log.LogConfig = {
-    handlers: {
-      default: new log.handlers.ConsoleHandler("DEBUG"),
-    },
-
-    loggers: {
-      default: {
-        level: "INFO",
-        handlers: ["default"],
-      },
-    },
-  };
-
-  if (options.debug) {
-    defaultLogLevel.loggers = {
-      default: {
-        level: "DEBUG",
-        handlers: ["default"],
-      },
+export function setupLogger(options: CmdGlobalOptions) {
+  if (!options.debug) {
```

Comment:
I like this, very simple. Have a look also at the mentioned ticket; the guys from deno mentioned you shouldn't rely on their `log` module anymore anyway, so it's good we did this change.

Reply:
ah great nice to hear.